common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -4,31 +4,34 @@
 @import views.html.fragments.atoms.mediaAtomCaption
 @import com.netaporter.uri.dsl._
 
-@(media: model.content.MediaAtom, displayCaption: Boolean, embedPage: Boolean, displayDuration: Boolean = true, displayEndSlate: Boolean = true)(implicit request: RequestHeader)
+@(media: model.content.MediaAtom, displayCaption: Boolean, embedPage: Boolean, displayDuration: Boolean = true, displayEndSlate: Boolean = true, isPlayable: Boolean = true)(implicit request: RequestHeader)
```

Comment:
could possibly call this just `playable` as the `is` part can be inferred from the type system.